### DISCUSSION TITLE
Tennis total games in the first set

### DISCUSSION THREAD

**Discussion by jbn1981** [cloudbet/docs#83] [Q&A]
Hi,

In tennis, what would be the marketUrl param for a request to total games in **set 1**?

I get a successful response when I query for total games in the **match** with this marketUrl:
```
tennis.total_games/over?total=21.5
``` 
But I can't find the correct marketUrl for total games just in set 1.

Tried this but no success:
```
tennis.total_games/over?total=9.5&period=set1`
```
I checked that the market exists, as you can see in this image of the event with id 23742668:

![total-games-set-1](https://github.com/cloudbet/docs/assets/32179115/792371de-bed2-4e4b-859b-8694b16c1e3d)

Thanks a lot!

**Comment by kgravenreuth** [comment]
Hi @jbn1981 

Apologies for the late reply. The Tennis Total Games in Set market is supported on the Cloudbet website, but not yet supported on the Cloudbet API. You can find the full set of supported Cloudbet API Markets here as mentioned in our API docs: https://gist.github.com/kgravenreuth/6703e1e213aecac4d5728f2f699d34e7#file-markets-json

If or when we add more markets, we will let you know. 

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Comment by kgravenreuth** [comment]
Hi @jbn1981 ,
Good news, we have now added more markets, including the `Tennis Total Games in Set` market to the Cloudbet API now. You can find the updated list of markets here: https://gist.github.com/kgravenreuth/6703e1e213aecac4d5728f2f699d34e7#file-markets-json

For your specific example, you would place a bet with a `marketUrl` as follows in the Place Bet Request:

`"marketUrl": "tennis.total_games_in_set/under?set=1&total=9.5"`

This information is available from the Feed API. Sample partial market response:

```json
      "submarkets": {
        "period=set1&period=wo": {
          "sequence": "158",
          "selections": [
            {
              "outcome": "under",
              "params": "set=1&total=9.5",
              "price": 2.274,
              "minStake": 0.1,
              "maxStake": 1609.2188,
              "probability": 0.421,
              "status": "SELECTION_ENABLED",
              "side": "BACK"
            },
```

Please let me know if you have any more questions. 

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Comment by jbn1981** [comment]
Many many thanks!
